pallets/loans/src/entities/pricing/external.rs

```diff
@@ -10,14 +10,41 @@ use frame_support::{self, ensure, RuntimeDebug, RuntimeDebugNoBound};
 use scale_info::TypeInfo;
 use sp_runtime::{
 	traits::{EnsureAdd, EnsureFixedPointNumber, EnsureSub, Zero},
-	DispatchError, DispatchResult, FixedPointNumber,
+	ArithmeticError, DispatchError, DispatchResult, FixedPointNumber,
 };
 
 use crate::{
 	entities::interest::ActiveInterestRate,
 	pallet::{Config, Error, PriceOf},
 };
 
+#[derive(Encode, Decode, Clone, PartialEq, Eq, TypeInfo, RuntimeDebugNoBound, MaxEncodedLen)]
+#[scale_info(skip_type_params(T))]
+pub struct ExternalAmount<T: Config> {
+	pub quantity: T::Rate,
+	pub settlement_price: T::Balance,
```

Comment:
Nit pick. Not sure if `settlement_price` is the right tearm for when repaying? cc @denniswell

Reply:
As a note: `ExternalAmount` is used for both: borrow & repay. So if changed there, it's changed for both.

Reply:
From my understanding of what this means, I think the name fits with its purpose.